v0/destinations/salesforce/networkHandler.js

```diff
@@ -0,0 +1,40 @@
+const {
+  proxyRequest,
+  prepareProxyRequest
+} = require("../../../adapters/network");
+const {
+  processAxiosResponse
+} = require("../../../adapters/utils/networkUtils");
+const { TRANSFORMER_METRIC } = require("../../util/constant");
+const { salesforceResponseHandler } = require("./utils");
+
+const responseHandler = (destinationResponse, destType) => {
+  const { status } = destinationResponse;
+  const message = `Request for destination: ${destType} Processed Successfully`;
+
+  salesforceResponseHandler(
+    destinationResponse,
+    "during Salesforce Response Handling",
+    TRANSFORMER_METRIC.TRANSFORMER_STAGE.RESPONSE_TRANSFORM,
+    destinationResponse?.rudderJobMetadata
+  );
+
+  return {
+    status,
+    message,
+    destinationResponse
+  };
+};
+
+class networkHandler {
```

Comment:
Are we able to identify salesforce's networkHandler with a class ? I don't think it is possible, can you verify @sauravlal15 ?

Reply:
yeah, I checked not able to do it.